src/fmu/sumo/sim2sumo/tables.py

```diff
@@ -229,6 +228,34 @@ def upload_tables(sim2sumoconfig, config, dispatcher):
         )
 
 
+def upload_vfp_tables_from_simulation_run(
+    datafile, options, config, dispatcher
+):
+    """Upload vfp tables from one simulator run to Sumo
+
+    Args:
+        datafile (str): the datafile defining the simulation run
+        options (dict): the options for vfp
+        config (dict): the fmu config with metadata
+        dispatcher (sim2sumo.common.Dispatcher): job dispatcher
+    """
+    logger = logging.getLogger(
+        __name__ + ".upload_vfp_tables_from_simulation_run"
+    )
+    keyword, vfp_tables = vfp_to_arrow_dict(datafile, options)
```

Comment:
Is `vfp_tables` misleading here?
I guess `vfp_to_arrow_dict` reads the "vfp" file into an arrow table. Is it then still a "vfp_table"?
Though I'm not certain what "vfp" or a "vfp table" is... Is it a file format?

Reply:
Vfp is a specific sort of datatype, in this case lift curves.. See earlier comment, but yes I agree, for the sake of the code it breaks the pattern. Have changed it now..